packages/react-components/src/messages/__tests__/Welcome.test.tsx

```diff
@@ -29,4 +29,30 @@ describe('welcome email template', () => {
     expect(mailToSupport.protocol).toBe('mailto:');
     expect(mailToSupport.pathname).toBe('techsupport@asap.science');
   });
+
+  it('renders different variants of the content', () => {
+    let cta1;
+    let cta2;
+    const { queryByRole, rerender } = result;
+
+    cta1 = queryByRole('link', { name: /activate account/i });
+    cta2 = queryByRole('link', { name: /create account/i });
+    expect(cta1).toHaveTextContent('Activate account');
+    expect(cta1).not.toBeNull();
+    expect(cta2).toBeNull();
+
+    rerender(
+      <Welcome
+        firstName="John Doe"
+        link="https://example.com"
+        variant={'InviteWelcomeTemplate'}
+      />,
+    );
+
+    cta1 = queryByRole('link', { name: /activate account/i });
+    cta2 = queryByRole('link', { name: /create account/i });
```

Comment:
```suggestion
```

I dont think you need these. i think the first two can be const. Not 100% though